Class-Code/Game of Life/pryan/sketch.js

```diff
@@ -1,14 +1,81 @@
-//Your Name(s)
-//Project Name
-//Date
+//Pat Ryan
+// Game of Life
+// March 2019
 
-/* A brief description of what the program is */
+// Conway's Game of Life Simulatio
 
+let current = [];
+ let cellSize = 10;
 
+ let columns,rows; 
 function setup() {
-    createCanvas(400, 400);
+    createCanvas(900, 500);
+    frameRate(10);
+
+columns = floor(width/cellSize); 
+rows = floor(height/cellSize);
+
+for(let row = 0; row < rows; row++){
+    current.push(new Array(columns).fill(0).map(()=>round(random())))
+    current.push(new Array(columns).fill(0))
```

Comment:
This looks a little fishy. Do you want to fill it with random 0's and 1's or with all 0's.